packages/nextlove/src/with-route-spec/middlewares/with-validation.ts

```diff
@@ -114,14 +119,56 @@ const zodIssueToString = (issue: z.ZodIssue) => {
   return `${issue.message} for "${issue.path.join(".")}"`
 }
 
+function validateJsonResponse<JsonResponse extends z.ZodTypeAny>(
+  jsonResponse: JsonResponse | undefined,
+  res: NextApiResponse
+) {
+  let shouldValidateResponsesBasedOnEnv = false
+  if (
+    (process.env.NEXTLOVE_VALIDATION_RESPONSES === undefined &&
```

Comment:
IIRC empty environment variables can be `""`

```suggestion
    (!Boolean(process.env.NEXTLOVE_VALIDATION_RESPONSES) &&
```